src/vaadin-date-time-picker.html

```diff
@@ -47,8 +93,204 @@
 
         static get properties() {
           return {
+            /**
+             * The value for this element.
+             *
+             * Supported date time format is based on ISO 8601 (without a time zone designator):
+             * - Minute precision `"YYYY-MM-DDThh:mm"`
+             * - Second precision `"YYYY-MM-DDThh:mm:ss"`
+             * - Millisecond precision `"YYYY-MM-DDThh:mm:ss.fff"` (default)
+             */
+            value: {
+              type: String,
+              notify: true,
+              value: '',
+              observer: '__valueChanged'
+            },
+
+            /**
+             * The current selected date time.
+             */
+            __selectedDateTime: {
+              type: Date
+            },
           };
         }
+
+        static get observers() {
+          return [
+            '__selectedDateTimeChanged(__selectedDateTime)'
+          ];
+        }
+
+        __parseDate(str) {
+          return customElements.get('vaadin-date-picker').prototype._parseDate(str);
```

Comment:
Afaik the idea was to keep the slotting API kind of private, so I don't think this is necessary (at least not). The reason to have the slots is so that `vaadin-date-time-picker-flow` can create a `DatePicker` and `TimePicker` on server side and slot those in (to be able to reuse their Java API easily).

Reply:
Hmm, nothing implies that slotting is going to be private. Forget about that and was a little bit confused. Probably we should add some docs about that. 

Can also be improved later.